view/statistics-files-pending.js

```diff
@@ -99,11 +100,22 @@ exports['statistics-main'] = function () {
 				function (data) { return data._rejected; }, this.processingStepsMeta,
 				"statistics-table-sub-header-sentback", function (data) { return data.rejected; }),
 
-			this.processingStepsMeta.frontDesk ?
+			getFrontDeskNames().size ?
```

Comment:
It would be cleaner to save the result of `getFrontDeskNames()` and reuse it in those two places